octobot_trading/constants.py

```diff
@@ -57,6 +57,7 @@
     "ENABLE_EXCHANGE_HTTP_PROXY_FROM_ENV", "True")
 ENABLE_CCXT_VERBOSE = os_util.parse_boolean_environment_var("ENABLE_CCXT_VERBOSE", "False")
 ENABLE_CCXT_RATE_LIMIT = os_util.parse_boolean_environment_var("ENABLE_CCXT_RATE_LIMIT", "True")
+MIN_TIME_BETWEEN_PUBLIC_WS_UPDATES = float(os.getenv("MIN_TIME_BETWEEN_PUBLIC_WS_UPDATES", "0.1"))  # avoid spamming CPU
```

Comment:
Do we really need this?

Reply:
I mean, with that code, by default, we will drop many data. I'm not sure that's what we want when we implement websockets